.artifact/tree.json

```diff
@@ -50843,6 +50843,230 @@
           }
         },
         "position_new": {}
+      },
+      "viam-office-02-22-2": {
```

Comment:
[note] I've left the old artifacts in for the present. I am planning to remove them after we've stabilized the frontend view but can delete them now if desired. Comment here if you'd like me to remove them now.